src/interactive.mjs

```
@@ -10,11 +10,26 @@ import {
   generateLoginLink,
   clearDrupalCache,
   gitUrlToGithubUrl,
-  extractPrNumber
+  extractPrNumber,
+  getGitBranches,
+  deployBranch
 } from './lagoon-api';
```

ESLint

[error] require file extension '.mjs'.

(n/file-extension-in-import)